Calendar

Deman4ig opened the pull request.

Deman4ig committed:
add solution for lesson-3 calendar

jsru commented:
Добавляю преподавателя (@GSemikozov) для код-ревью.

GSemikozov commented on 05-lection3/01-tooltip/README.md:
```diff
@@ -0,0 +1,5 @@
+# Тултип
```
системные файлы не следует добавлять в ПР - можно убрать

GSemikozov commented on 05-lection3/01-tooltip/test/solution.test.js:
```diff
@@ -0,0 +1,7 @@
+const expect = require('chai').expect;
```
еще один системный файл

GSemikozov commented on 05-lection3/02-modal/README.md:
```diff
@@ -0,0 +1,26 @@
+# Модалка
```
и еще

GSemikozov commented on 05-lection3/02-modal/index.js:
```diff
@@ -0,0 +1,19 @@
+(function() {
```
если скрипт не изменялся, то и его не стоит добавлять в ПР

GSemikozov commented on 05-lection3/02-modal/test/solution.test.js:
```diff
@@ -0,0 +1,7 @@
+const expect = require('chai').expect;
```
и еще один системный файл)

GSemikozov commented on 05-lection3/03-calendar/calendar.css:
```diff
@@ -0,0 +1,72 @@
+body {
```
стили которые не относятся к блоку стоит выносить наружу - либо в другой блок (если код будет переиспользоваться), либо в main (если это глобальные стили или сброс стилей), либо в тег style в html с решением задачи (если используется только для задачки)

GSemikozov commented on 05-lection3/03-calendar/calendar.css:
```diff
@@ -0,0 +1,72 @@
+    line-height: 24px;
+    display: flex;
+    justify-content: center;
+    color: #334D6E;
```
цвет нужно забрать из css переменных

GSemikozov replied:
как и везде ниже по коду (если такие цвета определены; если не определены и при этом они будут использоваться в других компонентах - можно расширить список переменных)

Deman4ig committed:
Delete README.md

Deman4ig committed:
Delete index.html

Deman4ig committed:
Delete solution.test.js

Deman4ig committed:
Delete modal.css

Deman4ig committed:
Delete tooltip.css

Deman4ig committed:
Delete README.md

Deman4ig committed:
Delete index.html

Deman4ig committed:
Delete index.js

Deman4ig committed:
Refactor calendar component

1. Added additional color variables
2. Moved generic styles out of component css

Deman4ig committed:
Merge branch 'master' of github.com:js-tasks-ru/htmlcss-20210513_deman4ig into calendar

Deman4ig committed:
Merge branch 'calendar' of github.com:Deman4ig/htmlcss-20210513_deman4ig into calendar
